crates/bevy_ui/src/layout_components.rs

```diff
@@ -115,12 +115,32 @@ pub enum Overflow {
     Hidden,
 }
 
+/// Defines if child UI items appear on a single line or on multiple lines
+#[derive(
+    Component, Copy, Clone, PartialEq, Eq, Debug, Default, Serialize, Deserialize, Reflect,
+)]
+#[reflect_value(PartialEq, Serialize, Deserialize)]
+pub enum Wrap {
```

Comment:
I'm not sure this should be moved outside fo `Flex`. If we want to add other layouts like CSS Grid, then it doesn't have any effect.

Reply:
I would expect this to be sharable with morphorm for example. The fact that CSS grid *doesn't* support wrapping makes it a bit of an oddity.

Fine to revert if others disagree.

Reply:
I think we can move this around later according to what makes sense as taffy gets more layout-algorithms online

Reply:
@plof27 agrees so I'm moving it back under.